widgets/blueprints/src/BlueprintsCatalog.tsx

```diff
@@ -135,54 +135,54 @@ export default function BlueprintsCatalog({
                             </Grid.Row>
                         )}
                     </Grid>
-
-                    <Grid.Column width="16">
-                        <div style={{ height: '80px' }} />
-                    </Grid.Column>
-                </DataSegment.Item>
-
-                {BlueprintActions.isCompleted(item) && (
-                    <div className="actionButtons">
-                        <Button
-                            icon="trash"
-                            content="Delete"
-                            basic
-                            labelPosition="left"
-                            onClick={event => {
-                                event.stopPropagation();
-                                onDeleteBlueprint(item);
-                            }}
-                        />
-                        {BlueprintActions.isUploaded(item) && (
-                            <>
-                                <Button
-                                    icon="rocket"
-                                    content="Deploy"
-                                    labelPosition="left"
-                                    onClick={event => {
-                                        event.stopPropagation();
-                                        onCreateDeployment(item);
-                                    }}
-                                />
-
-                                {!manager.isCommunityEdition() && widget.configuration.showComposerOptions && (
+                    {BlueprintActions.isCompleted(item) && (
+                        <Grid style={{ marginTop: 'auto', paddingTop: '1rem' }}>
+                            <Grid.Row className="centered">
+                                <Grid.Column className="centered aligned">
```

Comment:
Similar comment - can we use here:
```suggestion
                                <Grid.Column centered>
```
? 

I'm not sure about `aligned` class. Seems to me like it is added only when `textAlign` prop is provided. Do we need it?

Reply:
this is a bit tricky because if I want center buttons and I'm using "centered" on rows, Semantic UI gives Column "text-align: left" by default:
![Zrzut ekranu 2022-05-25 111605](https://user-images.githubusercontent.com/17828621/170227976-66d85d8a-a61a-4ce7-a53b-8c684328d579.png) however I might found a bit cleaner solution without className something like this:

``
```suggestion
   <Grid.Row centered>
     <Grid.Column textAlign="center">
```

Reply:
Looks good 👍🏻  Thx for checking that.